fastlane/Fastfile

```diff
@@ -55,4 +56,60 @@ platform :android do
       allow_nothing_to_commit: true
     )
   end
+
+  desc 'Builds a prototype build and uploads it to S3'
+  lane :build_and_upload_prototype_build do
+    UI.user_error!("'BUILDKITE_ARTIFACTS_S3_BUCKET' must be defined as an environment variable.") unless ENV['BUILDKITE_ARTIFACTS_S3_BUCKET']
+
+    comment_on_pr(
+      project: 'Automattic/Gravatar-SDK-Android',
+      pr_number: Integer(ENV.fetch('BUILDKITE_PULL_REQUEST', nil)),
+      reuse_identifier: 'app-prototype-build-link',
+      body: '🚧 Prototype builds will be available soon'
+    )
+
+    gradle(
+      task: ':demo-app:assemble',
+      build_type: 'debug'
+    )
+
+    upload_path = upload_to_s3(
+      bucket: 'a8c-apps-public-artifacts',
+      key: "gravatar-demo-prototype-build-#{generate_prototype_build_number}.apk",
+      file: lane_context[SharedValues::GRADLE_ALL_APK_OUTPUT_PATHS].first,
+      skip_if_exists: true
```

Comment:
Side note: `skip_if_exists` parameter is deprecated, use `if_exists: skip` instead.

Reply:
@wzieba Sorry for the typo here, that was supposed to be `if_exists: :skip` with a `:` in front of the `skip` keyword (to make it a Ruby symbol, instead of making it look like the name of a variable beinf referenced)